opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1397,7 +1397,7 @@ namespace Opm {
     template<typename TypeTag>
     ConvergenceReport
     BlackoilWellModel<TypeTag>::
-    getWellConvergence(const std::vector<Scalar>& B_avg, bool checkGroupConvergence) const
+    getWellConvergence(const std::vector<Scalar>& B_avg, bool checkWellGroupControls)
```

Comment:
I really do not like that this function is no longer const. What about storing the results (the boolean group_well_changed) in the class when called normally, and simply querying it here, instead of calling it all over again? Also, updateWellControls() is a bit overloaded already, with doing network pressure updates for example, I would be very careful here.

Reply:
fixed